packages/create-sitecore-jss/src/init-runner.ts

```diff
@@ -38,7 +46,7 @@ export const initRunner = async (initializers: string[], args: BaseArgs) => {
       }
     }
   };
-
+  prepareForProxy(initializers);
```

Comment:
For me it's a bit confusing why "prepareForProxy" is executed 2 times
from my understanding we just need to prepare the proxy build path a single time after all the prompts are answered.
Also "prepareForProxy" naming is confusing since it means that It can be applied for all the "proxy" apps, but here we care only about "node-xmcloud-proxy"